jdaviz/configs/imviz/tests/test_helper.py

```diff
@@ -1,6 +1,8 @@
 import numpy as np
+import pytest
 
 
+@pytest.mark.filterwarnings(r"ignore:The .* function is deprecated")
```

Comment:
the other option would be an if-statement in the for-loop (`if attr in (...): continue`).  That takes more work to keep updated since individual entries need to be added manually, but also could act as a list of things to remind us to remove them when the deprecation period is over.

Reply:
I made this explicit.